### PR TITLE
Modal for removing volunteers

### LINKED ISSUE #159: Remove volunteer modal
### Summary and Background
Remove volunteer modal confirms with admin if they want to unassign a volunteer.

### User Story
As an Admin, I want a confirmation before removing a volunteer in case I accidently select to remove a volunteer.

### Solution (Core Elements)
Create a modal that triggers when admin unassigns a volunteer.

The modal contains a cancel and remove button. Its text also contains the name of the volunteer to be removed.

### Mockups

Other designs:
- https://www.figma.com/file/TWJTNWjK75A6VgSisySTLe/Sistering-%2F-Admin-Experience?node-id=888%3A8374

### PR DESCRIPTION
**Pull request by hujoseph99**
## Ticket link

Closes #159

## Implementation description
* Created a modal for removing volunteers.

## Steps to test
1. Open the app
2. Go to Remove Volunteer Demo in menu bar
3. Enter a name of a volunteer you would like to remove
4. Verify that the modal works and looks as expected

## What should reviewers focus on?
* Styling of the modal matches mockup

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/admin/RemoveVolunteerModal.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import {
```
Comment (hujoseph99) [inline frontend/src/components/admin/RemoveVolunteerModal.tsx:1]: This file is the heart of the changes. All other changes are just for demo purposes and can be later removed.

**Review by LenaNguyen — CHANGES_REQUESTED** [review]
just a couple styling things!

File: frontend/src/components/admin/RemoveVolunteerModal.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      initialFocusRef={initialRef}
+    >
+      <ModalOverlay />
+      <ModalContent borderRadius={8} pt={3} pl={3} pr={3} pb={2}>
+        <ModalHeader textStyle="body-bold" marginBottom={0} pb={2}>
+          Remove volunteer?
+        </ModalHeader>
+        <ModalBody textStyle="body-regular" mt={0}>
+          Are you sure you want to remove {name}?
+        </ModalBody>
```
[the remark this answers is not in the document]
Comment (hujoseph99) [reply frontend/src/components/admin/RemoveVolunteerModal.tsx:42]: Fixed!

File: frontend/src/components/admin/RemoveVolunteerModal.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      isOpen={isOpen}
+      onClose={onClose}
+      isCentered
+      initialFocusRef={initialRef}
+    >
+      <ModalOverlay />
+      <ModalContent borderRadius={8} pt={3} pl={3} pr={3} pb={2}>
+        <ModalHeader textStyle="body-bold" marginBottom={0} pb={2}>
+          Remove volunteer?
+        </ModalHeader>
```
[the remark this answers is not in the document]
Comment (hujoseph99) [reply frontend/src/components/admin/RemoveVolunteerModal.tsx:37]: Fixed!

File: frontend/src/components/admin/RemoveVolunteerModal.tsx (hunk 1 above)
Comment (hujoseph99) [reply frontend/src/components/admin/RemoveVolunteerModal.tsx:1]: Fixed!

File: frontend/src/components/admin/RemoveVolunteerModal.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+  return (
+    <Modal
+      isOpen={isOpen}
+      onClose={onClose}
+      isCentered
+      initialFocusRef={initialRef}
+    >
+      <ModalOverlay />
+      <ModalContent borderRadius={8} pt={3} pl={3} pr={3} pb={2}>
```
[the remark this answers is not in the document]
Comment (hujoseph99) [reply frontend/src/components/admin/RemoveVolunteerModal.tsx]: Fixed!

**Review by LenaNguyen — DISMISSED** [review]

File: frontend/src/components/pages/Default.tsx [hunk 5]
```diff
@@ -180,6 +180,19 @@ const Default = (): React.ReactElement => {
...
+        />
+        <Button
+          text="Review Posting"
+          path={Routes.ADMIN_POSTING_CREATE_REVIEW_PAGE}
+        />
+        <Button text="Hooks Demo" path={Routes.HOOKS_PAGE} />
+        <Button
+          text="Remove Volunteer Demo"
+          path={Routes.REMOVE_VOLUNTEER_MODAL_DEMO}
+        />
```
Comment (LenaNguyen) [inline frontend/src/components/pages/Default.tsx:195]: "create shifts" and "review posting" may have been added in from the rebase. Can you remove all of these (including the demo)? They're not needed in main.

**Review by LenaNguyen — CHANGES_REQUESTED** [review]

File: frontend/src/constants/Routes.ts [hunk 6]
```diff
@@ -39,3 +39,6 @@ export const ADMIN_SCHEDULE_POSTING_PAGE = "/admin/schedule/posting/:id";
 
 export const VOLUNTEER_POSTING_AVAILABILITIES =
   "/volunteer/posting/:id/availabilities";
+
+// Temporary route for testing
+export const REMOVE_VOLUNTEER_MODAL_DEMO = "/remove_volunteer_demo";
```
[the remark this answers is not in the document]
Comment (hujoseph99) [reply frontend/src/constants/Routes.ts:44]: Do you want me to remove the full demo?

[the remark this answers is not in the document]
Comment (hujoseph99) [reply frontend/src/constants/Routes.ts]: Removed

**Review by LenaNguyen — APPROVED** [review]

File: frontend/src/components/pages/Default.tsx (hunk 5 above)
Comment (hujoseph99) [reply frontend/src/components/pages/Default.tsx]: Removed

**Pull request merged**